### PR TITLE
chore: prepare release 2024-03-19

### PR DESCRIPTION
**Pull request by Fluf22**
## Summary

This PR has been created using the `yarn release` script. Once merged, the clients will try to release their new version if their version has changed.

## Version Changes

- ~csharp: 7.0.0-alpha.5 (no commit)~
- ~dart: 1.6.1 (no commit)~
- ~go: 4.0.0-beta.8 (no commit)~
- ~java: 4.0.0-beta.26 (no commit)~
- ~javascript: 5.0.0-alpha.110 (no commit)~
- ~kotlin: 3.0.0-beta.21 (no commit)~
- ~php: 4.0.0-alpha.102 (no commit)~
- ~python: 4.0.0b11 (no commit)~
- ~ruby: 3.0.0.alpha.14 (no commit)~
- ~scala: 2.0.0-alpha.13 (no commit)~
- swift: 9.0.0-alpha.11 -> **`prerelease` _(e.g. 9.0.0-alpha.12)_**

### Skipped Commits

_(None)_

### CODE REVIEW

File: clients/algoliasearch-client-swift/CHANGELOG.md
Comment (shortcuts) [inline clients/algoliasearch-client-swift/CHANGELOG.md:1]: you might want to squash all changelogs and reset the version if it's the first real release

Comment (Fluf22) [reply clients/algoliasearch-client-swift/CHANGELOG.md:1]: I can do that manually and push here, without breaking anything?

Comment (shortcuts) [reply clients/algoliasearch-client-swift/CHANGELOG.md:1]: yup!

Comment (Fluf22) [reply clients/algoliasearch-client-swift/CHANGELOG.md:1]: nice.gif

**Review by shortcuts — APPROVED** [review]

**Pull request merged**